### PR TITLE
feat: removed unnecessary imports and deleted 1 unnecessary controller

### PR DESCRIPTION
**Pull request by nexusonez**

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -12,18 +12,6 @@
...
-        // Administrator admin = new Administrator("A0001", "Admin", "a01");
-
-        // JFrame frame = new JFrame();     
-        // frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-        // frame.setSize(1200, 700);
-
-        // ProjectView projectView = new ProjectView();
-        // ProjectController projectController2 = new ProjectController(lecturer2, projectView);
-        // frame.add(projectView.wrapper);
-        // frame.setVisible(true);
```
Comment (Ybzach) [inline Main.java:26]: pls dont delete the previous test, just comment it out, i dont wanna write this whole thing again lol

File: Model/CommentList.java [hunk 2]
```diff
@@ -78,11 +88,16 @@ public CommentModel getItem(String id) {
         }
         return null;
     }
+
     @Override
     public void setList() {
         // TODO Auto-generated method stub
```
Comment (Ybzach) [inline Model/CommentList.java]: try to use these methods to construct your commentList, there's a reason they're in the interface. Can refer to Huiyi's listModels(eg StudentList, LecturerList, etc) for simple reference of implementation.

File: Model/CommentList.java [hunk 3]
```diff
@@ -78,11 +88,16 @@ public CommentModel getItem(String id) {
...
         
     }
+    
+    /**
+     * A method that saves the commentList into the database comments.json
+     */
     @Override
     public void save() {
```
Comment (Ybzach) [inline Model/CommentList.java:104]: please avoid calling for Mapper, just use the JsonParser that you have already intialized.

**Review by Ybzach — CHANGES_REQUESTED** [review]

File: Model/CommentList.java (hunk 2 above)
Comment (nexusonez) [reply Model/CommentList.java]: resolved

File: Model/CommentList.java (hunk 3 above)
Comment (nexusonez) [reply Model/CommentList.java:104]: resolved

File: Main.java (hunk 1 above)
Comment (nexusonez) [reply Main.java:26]: resolved

**Pull request closed without merging**